src/components/App.js

```diff
@@ -56,12 +56,9 @@ const App = () => {
         <Route path="/cart" element={<Cart />}/>
         <Route path="/login" exact element={<Login currentUser={currentUser} setCurrentUser={setCurrentUser} token={token} setIsLoggedIn={setIsLoggedIn}/>}/>
         <Route path="/products" exact element={<Products currentUser= {currentUser} token={token} products={products} setProducts={setProducts}/>} />
-        <Route path="/products/:id" element={<SingleProduct token={token} products={products} setProducts={setProducts}/>} />
-        <Route path="/register" exact element={<Register setIsLoggedIn={setIsLoggedIn} setCurrentUser={setCurrentUser} currentUser= {currentUser} token={token} />}/>
-        <Route path="/account" exact element={<Account currentUser={currentUser} setCurrentUser={setCurrentUser} token={token} setIsLoggedIn={setIsLoggedIn}/>}/>
-        <Route path="/products/wines" element={<Wines token={token} products={products} setProducts={setProducts}/>}/>
-        <Route path="/products/cheeses" element={<Cheeses token={token} products={products} setProducts={setProducts}/>}/>
-        <Route path="/products/productpairs" element={<ProductPairs token={token} products={products} setProducts={setProducts}/>}/>
+        <Route path="/products/:id" exact element={<SingleProduct token={token} products={products} setProducts={setProducts}/>} />
```

Comment:
For register and account, wouldn't we want currentUser and token to be passed in so that they would be automatically navigated out of these areas? Users who are logged in shouldn't have access to the register page in exchange for their account page.

Reply:
I didn't touch the register and account routes for this PR.